docs/narr/cookiecutters.rst

```diff
@@ -1,11 +1,20 @@
-.. _cookiecutters:
+.. index::
+    single: cookiecutter
+
+.. _pyramid-cookiecutters:
```

Comment:
In a previous commit I removed the cookiecutter intersphinx link. I'd probably say you keep naming this `cookiecutters` instead of renaming to `pyramid-cookiecutters`.

Reply:
Oh, this is the problem: https://github.com/sphinx-doc/sphinx/issues/3921

I removed that second underscore on line 9 in an earlier commit, thinking it was a typo, but it actually has purpose. That in turn caused me to rename the reference to avoid these errors:

```
pickling environment... /Users/stevepiercy/projects/hack-on-pyramid/docs/narr/cookiecutters.rst:7: WARNING: Duplicate explicit target name: "cookiecutters".

...snip...

/Users/stevepiercy/projects/hack-on-pyramid/docs/narr/project.rst:76: WARNING: undefined label: cookiecutters (if the link has no caption the label must precede a section header)
/Users/stevepiercy/projects/hack-on-pyramid/docs/quick_tutorial/ini.rst:135: WARNING: undefined label: cookiecutters (if the link has no caption the label must precede a section header)
```

I restored both the name and double underscore, and all is well. Thanks!